src/main/java/menu/GeneralMenu.java

```diff
@@ -20,6 +21,9 @@ public void displayGeneralMenu (){
             case 2:
                 displayXMLMenu();
                 break;
+            case 3:
+                displayJsonMenu();
+                break;
             default:
                 System.out.println("Invalid option, try again");
```

Comment:
You should try not to use System.out.println in your projects, because this is unacceptable when working on a project. You can use a logger

Reply:
Got it. Thanks!